### DISCUSSION TITLE
Feature request - slashcommands should run on docs, developer, etc servicenow sites

### DISCUSSION THREAD

**Discussion by VanWeapon** [arnoudkooi/SN-Utils#281] [Ideas]
I use the slashcommand functionality to open up the docs and api pages all the time, I've added some of my own commands which open up the SN support site and some others, it would be neat to have a whitelist (maybe user-configurable?) of domains where the slashcommand function still works instead of only SN instances. 

I would use this to open up docs, api, or SN support search queries from other docs or SN sites without having to jump back to my instance first. 

This is really just a QOL thing so no big deal if it breaks stuff or is too annoying to retrofit

**Comment by arnoudkooi** [comment]
This can be done with the On-Prem version...
I am thinking of adding servicenow.com, which would trigger a popup / disable it for all users.
Also, there may become optional permissions for extensions in the future.

I will move this to a discussion and update when appropriate.
